kotlin/feature-layer-query/src/main/java/com/esri/arcgisruntime/sample/featurelayerquery/MainActivity.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.featurelayerquery
+
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import android.app.SearchManager
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.SearchView
+import com.esri.arcgisruntime.concurrent.ListenableFuture
+import com.esri.arcgisruntime.data.FeatureQueryResult
+import com.esri.arcgisruntime.data.QueryParameters
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import kotlinx.android.synthetic.main.activity_main.*
+import java.lang.Exception
+import java.util.Locale
+
+class MainActivity : AppCompatActivity() {
+
+  private val serviceFeatureTable: ServiceFeatureTable by lazy {
+    ServiceFeatureTable(getString(R.string.us_daytime_population_url))
+  }
+
+  private val featureLayer: FeatureLayer by lazy {
+    FeatureLayer(serviceFeatureTable)
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    featureLayer.apply {
+      opacity = 0.8f
+      maxScale = 10000.0
+    }
+
+    mapView.apply {
+      map = ArcGISMap(Basemap.createTopographic())
+    }.run {
```

Comment:
This is the pattern I tried when I first started with Kotlin and I got alot of feedback about readability when we starting nesting scoping functions. Possible alternative:

```
    mapView.map = ArcGISMap(Basemap.createTopographic()).copy().apply { 
      operationalLayers.add(featureLayer)
      initialViewpoint = Viewpoint(Point(-11000000.0, 5000000.0, SpatialReferences.getWebMercator()), 100000000.0)
    }
```